clusterloader2/testing/density/100_nodes/constraints.yaml

```diff
@@ -0,0 +1,24 @@
+coredns:
+  cpuConstraint: 0.05
+  memoryConstraint: 31457280 #30 * (1024 * 1024)
+fluentd-gcp:
+  cpuConstraint: 1
+  memoryConstraint: 241172480 #230 * (1024 * 1024)
+heapster:
+  cpuConstraint: 0.15
+  nemoryConstraint: 367001600 #350 * (1024 * 1024)
+kube-apiserver:
+  cpuConstraint: 1.7
+  memoryConstraint: 1048576000 #1000 * (1024 * 1024)
+kube-controller-manager:
+  cpuConstraint: 0.9
```

Comment:
I think this one got lost. Why this is higher than it used to be before?

Reply:
Probably we should understand that... it seems strange that it is higher, given that we didn't observe any issues with that in current tests where this is set to 0.8.

Reply:
I agree. I added issue #326.
Nevertheless I think this should not be a blocker for this PR.

Reply:
debatable, but ok.